try
	raise StackError
	print ('oops - didnt raise')
	return 1
rescue SXError error
	print ('ok, caught it')
	return 0
end

print ('oops - didnt catch')
return 1
